lib/config/migration.spec.ts

```diff
@@ -727,4 +723,27 @@ describe(getName(), () => {
     expect(isMigrated).toBe(true);
     expect(migratedConfig).toEqual({ extends: ['local>org/renovate-config'] });
   });
+  it('it migrates gradle-lite', () => {
```

Comment:
```suggestion

  it('it migrates gradle-lite', () => {
```

Reply:
This is already committed to v26, so I prefer not to add a new commit just for it and mess up the already large branch